app/repositories/stage-configuration-repository.js

```diff
@@ -0,0 +1,27 @@
+const { models } = require('../data')
+/**
+ * Get stage configuration by id
+ * @param {integer} id
+ * @returns stage configuration object
+ */
+async function get (id) {
```

Comment:
Remove the id parameter and return all data. This will be used to start our development and can be changed if needed.